### PR TITLE
ROX-23688: Update compliance routing

### PR DESCRIPTION
**Pull request by bradr5**
## Description

The compliance "Coverage" and "Schedules" sections should each have their own sub-navigation links under "Compliance 2.0"

Included:
- Clean up the the `routePaths` file, only compliance base path and sub navigation routes should be included here
- Add an individual routes file to each sub nav directory named `compliance.{directory}.routes.ts`. Initially considered adding these routes to the route handler file but ended up with a circular dependency – something that might happen often with that structure.

Note: route changes to the "status" section were left untouched since this section will be going away
 
## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

Manually tested routes and links

Side navigation:
Screenshot 2024-04-22 at 1 14 55 PM

### CODE REVIEW

File: ui/apps/platform/cypress/integration/compliance-enhanced/ComplianceEnhanced.helpers.js [hunk 1]
```diff
@@ -5,9 +5,8 @@ import { visit } from '../../helpers/visit';
 
 export const basePath = '/main/compliance-enhanced';
 export const statusDashboardPath = `${basePath}/status`;
-export const clusterCompliancePath = `${basePath}/cluster-compliance`;
-export const clusterComplianceCoveragePath = `${clusterCompliancePath}/coverage`;
-export const clusterComplianceScanConfigsPath = `${clusterCompliancePath}/scan-configs`;
+export const clusterComplianceCoveragePath = `${basePath}/coverage`;
+export const clusterComplianceScanConfigsPath = `${basePath}/schedules`;
```
Comment (pedrottimark) [inline ui/apps/platform/cypress/integration/compliance-enhanced/ComplianceEnhanced.helpers.js:9]: What do you think about same names as in routePaths.ts file?
* `complianceEnhancedCoveragePath`
* `complianceEnhancedSchedulesPath`

We might also brainstorm pro and con of with or without `Enhanced` especially within tests folder.

Comment (bradr5) [reply ui/apps/platform/cypress/integration/compliance-enhanced/ComplianceEnhanced.helpers.js:9]: Good catch, those routes should be what you suggested. Yeah I'm still not a huge fan of the word `Enhanced` being thrown around everywhere in compliance v2. I guess having the keyword in the file name makes it clear enough as to what we're dealing with.

File: ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ClustersCoverageTable.tsx [hunk 2]
```diff
@@ -72,7 +70,7 @@ function ClustersCoverageTable() {
                 <Tr key={cluster.clusterId}>
                     <Td dataLabel="Cluster">
                         <Link
-                            to={generatePath(complianceEnhancedCoverageClustersPath, {
+                            to={generatePath(clusterDetailsRoute, {
```
Comment (pedrottimark) [inline ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ClustersCoverageTable.tsx:73]: An observation, not necessarily a change request: my eyes stopped to look twice at the contrast
* name from within container `clusterDetailsRoute`
* name from outside container `complianceEnhancedSchedulesPath`

**Review by pedrottimark — APPROVED** [review]
One comment for your mental slow cooker as we move forward.

File: ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ClustersCoverageTable.tsx (hunk 2 above)
Comment (bradr5) [reply ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ClustersCoverageTable.tsx:73]: I think with "complianceEnhanced" being so long, I've mistakenly shy'd away from using it in certain instances.

**Pull request merged**